README.md

```diff
@@ -1,5 +1,7 @@
 # Rails Girls Guides
```

Comment:
maybe try girls streamline guides? im new and don't even know if I should be making suggestions here